SCClassLibrary/Common/Control/Buffer.sc

```diff
@@ -401,18 +401,25 @@ Buffer {
 	}
 
 	get { arg index, action;
-		OSCpathResponder(server.addr, ['/b_set', bufnum, index], { arg time, r, msg;
-			action.value(msg.at(3)); r.remove }).add;
-		server.listSendMsg(["/b_get", bufnum, index])
+		OSCFunc({ |message|
+			// The server replies with a message of the form [/b_set, bufnum, index, value].
+			// We want "value," which is at index 3.
+			action.value(message[3]);
+		}, \b_set, server.addr, argTemplate: [bufnum, index]).oneShot;
+		server.listSendMsg(["/b_get", bufnum, index]);
```

Comment:
Yeah, I was thinking about that.. `\foo` is more readable, since it visually resembles `/foo`. I think it might be more performant too, anyone know if that's true?